.github/workflows/ecosystem-recompile.yml

```diff
@@ -0,0 +1,43 @@
+name: Ecosystem | Recompile members.json
+
+on:
+  push:
+    branches:
+      - 'main'
+  workflow_dispatch:
+
+
+jobs:
+    runs-on: ubuntu-latest
+    env:
+      tox_env: "py39"
+      python-version: "3.9"
+    steps:
+    # setup deps
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ env.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ env.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
+        pip install -r requirements-dev.txt
+
+    - name: Run tests
+      run: |
+        tox -epy39
```

Comment:
We already have this in the `tests.yml` workflow
```suggestion

```

Reply:
Ok sounds good :)